force-app/main/default/classes/ClassesObjectsMethods.cls

```diff
@@ -26,184 +26,164 @@
-        
-        // Return the list with the string data
-        // In the first position add the string length
-        // In the second position add the substring
-        // In the third position add the string in lower case
+        String helloSalesforce = 'Hello, Salesforce!';
+        Integer strLength = helloSalesforce.length();
+        String strSubString = helloSalesforce.substring(0, 5);
+        String strLower = helloSalesforce.toLowerCase();
 
-        return null; // Replace null with the variable you used to store the result
+        List <String> helloList = new List<String>();
+
+        helloList.add(String.valueOf(strLength));
+        helloList.add(strSubString);
+        helloList.add(strLower);
+
+        return helloList;
     }
 
     /**
-     * Question 2
      * Check if the input string starts with 'Sales', ends with 'Force', and contains 'Awesome'.
      * Example: checkStringContent('Salesforce is Awesome!') should return a list like [true, false, true].
      * @param input The input string.
      * @return A list of boolean values indicating whether the string starts with 'Sales', ends with 'Force', or contains 'Awesome'.
      */
     public static List<Boolean> checkStringContent(String input) {
-        // Initialize a list to store the results
-        
-        // Check if the string starts with 'Sales'
-        
-        // Check if the string ends with 'Force'
-        
-        // Check if the string contains 'Awesome'
-        
-        return null; // Replace null with the variable you used to store the result
+        List<Boolean> result = new List<Boolean>();
+
+        Boolean startsWith = input.startsWith('Sales');
+        result.add(startsWith);
+
+        Boolean endsWith = input.endsWith('Force');
+        result.add(endsWith);
+
+        Boolean contains = input.contains('Awesome');
+        result.add(contains);
+    
+        return result; 
     }
 
     /**
-     * Question 3
      * Take a sentence then remove any duplicate words, and return the sentence with only unique words.
-     * Example: removeDuplicateWords('Hello Hello Salesforce Salesforce') should return 'Hello Salesforce'.
+     * Example: removeDuplicateWords('Hello, Hello, Salesforce!') should return 'Hello, Salesforce!'.
      * @param sentence The input sentence.
      * @return A string consisting of only unique words from the input sentence.
      */
     public static String removeDuplicateWords(String sentence) {
-        // Split the sentence into words
-
-        // Create a set to store the unique words
-
-        // Add each word to the set
-
-        // Join the unique words back into a sentence
-
-        // Return the sentence with unique words
-        return null; // Replace null with the variable you used to store the result
+        List<String> newList = new List<String>();
+        newList = sentence.split(' ');
+        Set<String> newSet = new Set<String>(newList);
```

Comment:
Nice job converting a list to a set in one line of code (most students have been using a loop here).